client/src/www/views/root_view/add_access_key_dialog/index.ts

```diff
@@ -123,7 +123,6 @@ export class AddAccessKeyDialog extends LitElement {
     accessKey: string,
     validate: (accessKey: string) => Promise<boolean>
   ) {
-    this.isValidAccessKey = false;
```

Comment:
What will hapoen if you paste a valid key first, and then modify it to be an invalid key. The validation will take some time since it's async, will the button be enables for a short period of time then?

Reply:
@jyyi1 this is the fix you suggested to fix the issue in my PR. The button will be enabled for a short time, but it's not a big deal, since the add will fail anyway.

@daniellacosse could you clarify what the infinite loop is? `updateIsValidAccessKey` is only called on change of the text field.